.github/workflows/validate_pr_labels_and_release_notes.yml

```
@@ -14,6 +18,7 @@ jobs:
           fetch-depth: 0
 
       - name: Get the list of allowed pull request labels
+        id: get-allowed_labels
```

Just to make it look nicer

```suggestion
        id: get-allowed-labels
```